packages/client/src/components/EmailSettingsBanner.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+    <div>
+      <b-alert style="display: flex;  text-align: center; max-width: 1100px;" variant="success" show v-model="showBanner" dismissible>
+          <b-icon icon="envelope-fill" font-scale="1.5" style="width: 50px; color: #0FA958;"></b-icon>
+          <div style="padding-right: 10px; color: #212529;">
+          <b>The Grant ID Tool now has email notifications!</b>
+          Fine tune which email you will recieve in <u v-on:click="showProfileSettings()">Settings</u> or turn on all notifications here:
+        </div >
+        <b-form-checkbox v-model="checked" name="email-opt-in-switch" switch @change="onUserSubscriptionChangeSubmit"/>
+      </b-alert>
+      <b-alert
+        id="email-settings-banner"
+        ref="modal"
+        title="Email Settings Banner"
+        @show="resetModal"
+        @hidden="resetModal"
+        ok-only
+      >
+      </b-alert>
+    </div>
+  </template>
+
+<script>
+import { mapActions, mapGetters } from 'vuex';
+
+export default {
+  props: {
+    showBanner: Boolean,
+    showProfileSettings: Function,
+  },
+  data() {
+    return {
+      checked: true,
```

Comment:
Does `checked: true` indicate to the user that they are already subscribed to all notifications? if so, we may want to flip it so the user has an option to subscribe to all notifications.

Reply:
right now, by default the user is not subscribed to any notification. So the banner should show up as `checked: false` initially

Reply:
Ah sorry, ok will update